src/assets/scss/components/DicomImageViewer.scss

```diff
@@ -108,14 +108,14 @@ $opacityLevel: 0.75;
 }
 
 .expandedBottom {
+  height: 15em;
```

Comment:
I wonder if `13em` would work better? On my display, it seems like there is some extra unnecessary space at the bottom.
![Screenshot from 2021-03-23 23-44-36](https://user-images.githubusercontent.com/25886657/112251550-dc2aa300-8c31-11eb-8a3b-699637f47bdc.png)